pkg/printers/junitxml.go

```diff
@@ -39,11 +41,15 @@ type failureXML struct {
 }
 
 type JunitXML struct {
-	w io.Writer
+	extended bool
+	w        io.Writer
```

Comment:
why not 

```go
type JunitXMLExtended struct {
    JunitXML
}
```

or inheritance no make sense here?

Reply:
It is not really useful in this context: the "extended" coded is inside a loop, I will not create a base struct that takes a function as a parameter (to use it inside the loop) + another struct + a method/function, just to avoid using a boolean.